content/en/docs/pluggable-components/kubeedge.md

```diff
@@ -107,4 +107,30 @@ iptables-hphgf                                    1/1     Running   0          5
 
 {{</ tab >}}
 
-{{</ tabs >}}
\ No newline at end of file
+{{</ tabs >}}
+
+## Custom Configurations
+
+After you enable KubeEdge, you can manually create a [ConfigMap](../../project-user-guide/configuration/configmaps/) to customize some configurations, such as the download URL of `keadm` and the version of KubeEdge. Local configurations will be dynamically updated based on the ConfigMap.
+
+Here is an example of the ConfigMap:
+
+```yaml
+apiVersion: v1
```

Comment:
This sample could be shorter

```yaml
apiVersion: v1
data:
  region: zh
  url: ""
  version: v1.6.1 # The default installed version of KubeEdge.
kind: ConfigMap
metadata:
  name: edge-watcher-config
  namespace: kubeedge
```
What is the default URI if not set?
And what are the valid values for the URI? @zhu733756

Reply:
The truely field should be `uri`, not `url`.  This field `uri`  is used to support a download url for custom keadm binary package. The valid values can be set as below:
```
region: zh or en
uri: https://kubeedge.pek3b.qingstor.com/bin/v1.6.1/$arch/keadm-v1.6.1-linux-$arch.tar.gz  && tar xvf keadm-v1.6.1-linux-$arch.tar.gz
version: kubeedge version(v1.6.1)
```
It is not recommended to set custom download binary because there are other related files to be set. However, we support this field for personalized compilation requirements.

Reply:
Changed. BTW, if my memory serves, Google instances have troubling downloading QingStor links. If possible, we need to test if a KubeSphere cluster on GCP can enable KubeEdge for the default value of region is set to `zh`. Otherwise, GCP users will have to create the ConfigMap and set the region manually.

Reply:
Yeah, it is recommended to set region to en by GCP users.

Reply:
Why we add a URI param here because it's automatically choosed between QingStor and github depending on the region value `en` or `cn`

More params could confuse uses with little benefits

@zhu733756

Reply:
It was optional configuration by API Scheme Design that we have disscused. If we want to make clearly, we can remove the introduce from the document.

Reply:
I think we can introduce the parameters in the cmd returned by the node-join api. Tell user the region in the cmd can change, instead of applying the cm here.

Reply:
No need to expose a URI param, we'd better remove it from the installer and doc.
And no need to introduce more params in the node-join command line. Using region is enough for Chinese and foreign users.

@zhu733756 @Sherlock113

Reply:
Ok I see. So perhaps I can delete the whole part Custom Configurations in the installation guide, and add this part in a separate FAQ guide. For most users, I don't think they care how KubeEdge is downloaded as long as they can use the function. If users want to customize these parameters, we just tell how to do it. @zhu733756 @benjaminhuo How about this?

Reply:
@Sherlock113 yes, pls go ahead.

@zhu733756 Would you remove the URI from the installer as well

Reply:
Yeah. By the way, if the document do not mention it. The code maybe not need to change.

Reply:
It'll be confusing if users find out there is something undocumented in the guide.
So just removing it from installer is ok